readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Bezushk0.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Bezushk0.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR descriptio
<img width="969" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/bfe9c97b-3912-453a-a94c-d86110e1c2a5">